scripts/EN/Harry

```diff
@@ -5690,18 +5690,18 @@ Area(0xca); // 0x72d8: 00001026
 CleanScreen(); // 0x72de: 00000064
 LoadBackground(0xaf); // 0x72e2: 00001069
 DrawBackground(); // 0x72e8: 00000065
-ShowText((1596)"At that time I remembered. I searched my pockets...got it. The pet bottle. The vermilion liquid is inside it. I scooped it up in the hospital."); // 0x72ec: 0000100c
+ShowText((1596)"At that time I remembered.¥nW2I searched my pockets…W2got it.¥nW2The plastic bottle.¥nW2The vermilion liquid is inside it.¥nW2I scooped it up in the hospital."); // 0x72ec: 0000100c
 HandleInput(); // 0x72f2: 00000010
-ShowText((1597)"The red of Cybil's eyes looked a lot like the red of that liquid. I hold up the pet bottle so that Cybil can see it and shake it left and right."); // 0x72f6: 0000100c
+ShowText((1597)"The red of Cybil's eyes looked a lot like the red of that liquid.¥nW2I hold up the bottle W1so that Cybil can see it W1and shake it left and right."); // 0x72f6: 0000100c
 HandleInput(); // 0x72fc: 00000010
-ShowText((1598)"And then just for a moment her expression changed to one of imploring... She must be aware. But she seems to be caught on an edge in her body, her body that will not do her bidding."); // 0x7300: 0000100c
+ShowText((1598)"And then just for a moment W1her expression changed to one of imploring…¥nW2She must be aware.¥nW2But she seems to be caught on an edge in her body, her body that will not do her bidding."); // 0x7300: 0000100c
 HandleInput(); // 0x7306: 00000010
-ShowText((1599)"It is like she is being attracted to me... With her gun ready, Cybil approaches. Upon closer inspection I can see that her eyes are full of tears."); // 0x730a: 0000100c
+ShowText((1599)"It is like she is being attracted to me…¥nW2With her gun ready, Cybil approaches.¥nW2Upon closer inspection I can see that her eyes are full of tears."); // 0x730a: 0000100c
 HandleInput(); // 0x7310: 00000010
-ShowText((1600)"I unscrew the cap of the pet bottle. No one is riding the wooden horses, so their speed increases. I think that it is like time is rapidly passing for everything but the two of us."); // 0x7314: 0000100c
+ShowText((1600)"I unscrew the cap from the bottle.¥nW2No one is riding the wooden horses, so their speed increases.¥nW1I think that it is like time is rapidly passing for everything but the two of us."); // 0x7314: 0000100c
```

Comment:
Here I just removed plastic because it felt redundant at this point.

Feel free to say "bottle" or "plastic bottle" as it works best in your language.